x-pack/plugins/stack_alerts/server/alert_types/geo_containment/geo_containment.ts

```diff
@@ -101,9 +101,16 @@ export function getActiveEntriesAndGenerateAlerts(
   allActiveEntriesMap.forEach((locationsArr, entityName) => {
     // Generate alerts
     locationsArr.forEach(({ location, shapeLocationId, dateInShape, docId }) => {
+      let entityDateTime;
+      if (dateInShape) {
+        // Handle epoch millis
+        entityDateTime = new Date(isNaN(dateInShape) ? dateInShape : +dateInShape).toISOString();
```

Comment:
`new Date` can return an "Invalid data" instance. Should this be checked and the date field set to null if the Date is invalid?